Reset

jn167 opened the pull request:
password recovery, account email verification, and account email domain validation

jn167 committed:
password reset

Attempted to create password recovery feature

jn167 committed:
R1.2.1 & R1.2.2

account email verification and account email domain validation implemented

jholmes-dev reviewed (approved):
Good work. Merge staging into this branch and do some local integration testing before merging.

jn167 committed:
Merge branch 'staging' into Reset

jn167 committed:
Password recovery (R1.3),  Account email verification (R1.2.1),  Account email domain validation (R1.2.2)

Password reset page
Password recovery/change password
Account email verification
Account email domain validation
Email service provider setup to handle email validation and reset password requests

(minor changes commenting)

jn167 committed:
messenger

Live messenger implementation for seller and buyer messaging
